### PR TITLE
LTI 1.1 `/config_xml` view internal rename

### PR DESCRIPTION
**Pull request by marcospri**
Moving the existing /config_xml view to the canvas namespace. Including the version in the route to accommodate the v13 in a follow up PR.

### Testing

Just open http://localhost:8001/config_xml

### CODE REVIEW

File: lms/routes.py [hunk 1]
```diff
@@ -114,3 +113,5 @@ def includeme(config):
     config.add_route(
         "lti.v11.deep_linking.form_fields", "/lti/1.1/deep_linking/form_fields"
     )
+
+    config.add_route("lti.v11.config", "/config_xml")
```
Comment (marcospri) [inline lms/routes.py:117]: Moving it down here with the rest of LTI related routes.

File: lms/views/lti/config.py [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 
 @view_config(
-    route_name="config_xml",
+    route_name="lti.v11.config",
```
Comment (marcospri) [inline lms/views/lti/config.py:5]: Following the convention of other LTI names.

**Review by jon-betts — APPROVED** [review]
This only changes the internal name of the view and it's location. The external URL will remain the same. This is a bit of a shame, but I think might be unavoidable, as this URL has already been passed out to customers when they integrated us.

I'm not sure if it's ever read again, but it's certainly safe to leave it alone.

**Review by jon-betts — APPROVED** [review]
Yep this looks good.

**Pull request merged**